client/websocket.go

```diff
@@ -190,6 +190,10 @@ func (c *Client) handleWSMsg(msg ws.Message) error {
 			}
 			var recState CallJobState
 			recState.FromMap(data)
+			c.emit(WSCallJobState, recState)
+
+			// Below is deprecated as of v0.14.0, kept for compatibility with earlier versions
+			// of transcriber
 			if recState.Type != "recording" {
```

Comment:
@streamer45 Sorry, I'm not really sure what you mean here -- please feel free to make the changes, I'm totally fine with whatever.

Reply:
It's approved, just merge it :p 

I simply meant the following:

```go
if recState.Type == "recording" {
  c.emit(WSCallRecordingState, recState)
}
```

Mostly to make that the special case rather than the default.

Reply:
Ah, now I get what you meant.